### PR TITLE
feat: record all calls

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

**Comment by davehorton** [comment]
Could you start an opentelemetry span when we start the recording (background listen), and close it when we stop the listen/recording?

File: app.js [hunk 1]
```diff
@@ -20,7 +20,12 @@ const tracer = require('./tracer')(JAMBONES_OTEL_SERVICE_NAME);
 const api = require('@opentelemetry/api');
 srf.locals = {...srf.locals, otel: {tracer, api}};
 
-const opts = {level: JAMBONES_LOGLEVEL};
+const opts = {
+  level: JAMBONES_LOGLEVEL,
+  timestamp: () => {
```
Comment (davehorton) [inline app.js:25]: Hmm, interesting,.  Why is this needed? I removed calls to timestampt like this a while ago because there is a cost in terms of performance - probably only noticed at very high call loads, but still

Comment (xquanluu) [reply app.js:25]: Hi Dave, about the timestamp, sometime I need to calculate how much time between 2 event, but yes, If it's impact the performance, I will remove it after finish debuging
maybe I will put a flag there to enable this feature instead of completely remove it


when debuging by log, timestamp is quite important to map the event with other tool like pcap and another server log

Comment (davehorton) [reply app.js]: yes, keep in mind when looking at a log file you can pipe it through pino-pretty to get timestamps, like this
```
cat ~/.pm2/log/jambonz-feature-server.log | pino-pretty -c -t 
```

Comment (xquanluu) [reply app.js]: as now there is pino-pretty, I removed this code.

**Comment by davehorton** [comment]
testing on jambonz.me with a fresh install of the branches for this PR and I get an error
```
{"level":30,"time":1685556763577,"pid":28919,"hostname":"ip-10-0-53-191","callId":"93496eb5-7a81-123c-d0b5-021e3c815c57","callSid":"62c49ff7-11a8-4602-b26b-98e032e7a6de","accountSid":"9351f46a-678c-43f5-b8a6-d4eb58d131af","callingNumber":"+15083084809","calledNumber":"+15083728299","traceId":"0b7761a8e84c91dd488ac557a326e407","err":{"type":"Error","message":"auth: property username has invalid data type","stack":"Error: auth: property username has invalid data type\n    at validateVerb (/home/admin/apps/jambonz-feature-server/node_modules/@jambonz/verb-specifications/jambonz-app-json-validation.js:73:17)\n    at validateVerb (/home/admin/apps/jambonz-feature-server/node_modules/@jambonz/verb-specifications/jambonz-app-json-validation.js:99:9)\n    at makeTask (/home/admin/apps/jambonz-feature-server/lib/tasks/make_task.js:15:3)\n    at InboundCallSession.startBackgroundListen (/home/admin/apps/jambonz-feature-server/lib/session/call-session.js:514:35)\n    at InboundCallSession.enableRecordAllCall (/home/admin/apps/jambonz-feature-server/lib/session/call-session.js:1811:18)\n    at InboundCallSession._notifyCallStatusChange (/home/admin/apps/jambonz-feature-server/lib/session/call-session.js:1764:18)\n    at InboundCallSession.emit (node:events:513:28)\n    at InboundCallSession.propagateAnswer (/home/admin/apps/jambonz-feature-server/lib/session/call-session.js:1519:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async InboundCallSession._evalEndpointPrecondition (/home/admin/apps/jambonz-feature-server/lib/session/call-session.js:1385:9)"},"opts":{"verb":"listen","url":"undefined/record/aws_s3","wsAuth":{},"mixType":"stereo"},"msg":"CallSession:startBackgroundListen - Error creating listen task"}
```

**Review by davehorton — APPROVED** [review]

**Comment by davehorton** [comment]
please rebase to main

**Comment by xquanluu** [comment]
rebased.
